pytrip/tripexecuter/field.py

```diff
@@ -92,8 +77,6 @@ def _print(self):
         out += "   Field {:d} '{:s}'\n".format(self.number, self.basename)
         out += "----------------------------------------------------------------------------\n"
         out += "|  UUID                         : {:s}\n".format(str(self.__uuid__))
-        out += "|  Projectile                   : {:s}-{:s}\n".format(self.projectile,
```

Comment:
Can you add a printout of kernel attached to this field ?
Otherwise user won't have information what kind of projectile is connected to this field.

Reply:
> @ljelen can you make a test with an isotope which not the most abundant. For example I'd like to make a plan with deuterium (2H) as a projectile. Will it work ?
> 
> TRiP98 documentation says that it should be accepted:
> http://bio.gsi.de/DOCS/TRiP98/PRO/DOCS/trip98dataobjects.html#Projectiles
> 
> > Throughout TRiP98 ionic projectiles may be specified as
> > 
> > Valid specifications are e.g.:
> > C
> > 12C
> > 12C6+
> > 12C5+
> > 2H
> > Note that they are case-sensitive, that is
> > 8BE
> > would be an error, whereas
> > 8Be
> > would be OK. When the mass number is omitted, the most abundant nuclide is chosen.

Add another unit test based on test_exec.py, or edit it?

Reply:
I meant a manual test, not a unit test.